src/components/BarChartCard/barChartUtils.js

```diff
@@ -1,5 +1,6 @@
 import moment from 'moment';
 import isNil from 'lodash/isNil';
+import isEmpty from 'lodash/isEmpty';
```

Comment:
PR open for these changes: https://github.com/carbon-design-system/carbon-addons-iot-react/pull/1686